public/app/plugins/panel/geomap/layers/data/geojsonMapper.ts

```diff
@@ -77,6 +78,11 @@ export const geojsonMapper: MapLayerRegistryItem<GeoJSONMapperConfig> = {
     const vectorLayer = new VectorLayer({
       source,
       style: (feature: FeatureLike) => {
+        const type = feature.getGeometry()?.getType();
+        if (type === 'Point') {
+          return circleMarker({color:DEFAULT_STYLE_RULE.fillColor});
```

Comment:
current behavior is that nothing at all is shown for point data (load the airports to test)